runelite-client/src/main/java/net/runelite/client/plugins/ExternalPf4jPluginManager.java

```diff
@@ -146,11 +151,29 @@ public void loadPlugins()
 			{
 				if (!(e instanceof PluginAlreadyLoadedException))
 				{
+					String plugin = pluginPath.toString().substring(pluginsRoot.toString().length() + 1);
+					duplicatePlugins.add(plugin);
 					log.error("Could not load plugin {}", pluginPath, e);
 				}
 			}
 		}
 
+		if (duplicatePlugins.size() > 0)
+		{
+			for (String dupe : duplicatePlugins)
+			{
+				log.error("Duplicate plugin detected: {}", dupe);
```

Comment:
You should change this to be 1 log message, with all the plugins, rather than multiple log messages. Reuse the formatted variable you create after this